codegen/src/botocore.in.rs

```diff
@@ -92,6 +92,11 @@ impl Service {
     pub fn shape_type_for_member<'a>(&'a self, member: &Member) -> Option<&'a str> {
         self.shapes.get(&member.shape).map(|ref shape| &shape.shape_type[..])
     }
+
+    pub fn typed_errors(&self) -> bool {
+        self.service_type_name() == "Kinesis" ||
+        self.service_type_name() == "KinesisFirehose"
```

Comment:
Probably more idiomatic to do this with something like:

```rust
match self.service_type_name() {
    "Kinesis" | "KinesisFirehose" => true,
    _ => false,
}
```

Although I guess it doesn't make much difference.